src/_guides/language/effective-dart/design.md

```diff
@@ -230,9 +235,10 @@ Those kinds of members should be named using an imperative verb phrase that
 clarifies the work the member performs.
 
 {:.good-style}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (verb-for-func-with-side-effect)" replace="/;//g"?>
 {% prettify dart %}
-list.add()
-queue.removeFirst()
+list.add(...)
+queue.removeLast()
```

Comment:
Using `removeLast()` because it is an actual `Iterable` method name (of course, that assumes that is a dequeue).

Reply:
Right, I was looking at the `List` API. My objective was to stay within the scope of types offered by `dart:*` packages. I just noticed that [DoubleLinkedQueue](https://api.dartlang.org/stable/latest/dart-collection/DoubleLinkedQueue-class.html) is a part of `dart:collections`, and it has both remove methods as you mention. So I'll revert this change.